pro-duck-store/src/App.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './App.css';
+
+
+import MiniDrawer from "./components/Drawer/drawer";
+import {Store} from "./pages/store";
+import {BrowserRouter, Route, Switch} from "react-router-dom";
+import {Cart} from "./pages/cart";
+import {User} from "./pages/user";
+
+
+function App() {
+  return (
+      <BrowserRouter>
+      <MiniDrawer/>
+      <main style={{display: 'flex', paddingLeft: '80px'}}>
+          <Switch>
+              <Route exact path="/" render={() => <h1>Home page</h1>}/>
+              <Route path = "/store" component={Store}/>
```

Comment:
As far as I see this functionality of showing a list of products is not finished